apis/server/volume_bridge.go

```diff
@@ -41,12 +41,26 @@ func (s *Server) createVolume(ctx context.Context, rw http.ResponseWriter, req *
 		return err
 	}
 
-	volume := types.VolumeInfo{
-		Name:   name,
-		Driver: driver,
-		Labels: config.Labels,
+	volume, err := s.VolumeMgr.Get(ctx, name)
+	if err != nil {
+		return err
+	}
+
+	respVolume := types.VolumeInfo{
+		Name:       name,
+		Driver:     driver,
+		Labels:     config.Labels,
+		Mountpoint: volume.Path(),
+		CreatedAt:  volume.CreationTimestamp.Format("2006-1-2 15:04:05"),
+		Status:     make(map[string]interface{}),
+	}
+	for k, v := range volume.Options() {
```

Comment:
I think we can move the following codes above just before respVolume's initialization. We can use 
```
var status map[string]string
for k, v := range volume.Options() {
    if k != "" && v != "" {
        status[k] = v
    }
}
```

Then we can use this `status` in the `respVolume` initialization.

Although two ways can achieve the same effect, maybe we can make code more clear with two parts: data preparation and ``respVolume` initialization`. And this makes no separated assignments.

Reply:
ok